src/app/components/receive/receive.component.ts

```diff
@@ -38,9 +40,46 @@ export class ReceiveComponent implements OnInit {
     private util: UtilService) { }
 
   async ngOnInit() {
+    this.shouldLoop = true;
+    this.updateList();
     await this.loadPendingForAll();
   }
 
+  ngOnDestroy() {
+    // stop checking the list when not in this view
+    this.shouldLoop = false;
+  }
+
+  // check for new pending in any account every 2sec and update list if needed
+  async updateList() {
```

Comment:
Yeah it's a bit duplicate. I use a faster poll interval here because the list is more essential and what the user use when receiving. The account detail view is more general info. But sure, maybe some kind of service would be better. I'm not sure how to make it but I can try if you are ok with my other comments?